src/modules/menu/components/search/SearchBar.vue

```diff
@@ -20,16 +20,21 @@ const searchQuery = computed(() => store.state.search.query)
 const hasResults = computed(() => store.state.search.results.length > 0)
 const isPhoneMode = computed(() => store.getters.isPhoneMode)
 
-watch(hasResults, (newValue) => {
-    // if an entry has been selected from the list, do not show the list again
-    // because the list has been hidden by onEntrySelected. Also if the search bar is pristine (not
-    // yet modified by the user) we don't want to show the result (e.g. at startup with a swisssearch
-    // query param)
-    if (!selectedEntry.value && !isPristine.value) {
-        log.debug(`Search has result changed to ${newValue}, change the show result to ${newValue}`)
-        showResults.value = newValue
-    }
-})
+watch(
+    hasResults,
+    (newValue) => {
+        // if an entry has been selected from the list, do not show the list again
+        // because the list has been hidden by onEntrySelected.
```

Comment:
Why changing the old comment ?

Reply:
because i removed the && !isPristine.value from the if statement so now if it is pristine the results are shown
this is to show the results when the url has swisssearch in it